config/settings.yml

```diff
@@ -1,2 +1,14 @@
 plugins:
-  crowd_verbose_log: false
+  crowd_verbose_log:
+    default: false
+  crowd_validate_email:
+    default: false
+  crowd_groups_enabled:
+    default: true
```

Comment:
For backwards compatibility this should default to false. We don't want to surprise people.

Reply:
Good point. These were our settings.
For backward compatibility the crowd_validate_email should also be `true` (change made).